### PR TITLE
Issues 449, 521, 525, 526: Catch `develop-2.0.0` up to `develop`

### PR DESCRIPTION
**Pull request by ajnelson-nist**
This is part of resolving #449, #521, #525, and #526 .

# Coordination

- [x] Pull Request is against correct branch
- [x] Pull Request is in Draft status before CI has passed
- [x] CI passes in UCO feature branch against `develop-2.0.0`
- [x] Milestone linked

### CODE REVIEW

**Comment by ajnelson-nist** [comment]
This is ready to be merged.

File: src/create-catalog-v001.xml.py
Comment (plbt5) [inline src/create-catalog-v001.xml.py:102]: There's no `else` clause catching an empty graph. Is that a non-fatal error?

**Review by plbt5 — CHANGES_REQUESTED** [review]

File: src/create-catalog-v001.xml.py
Comment (ajnelson-nist) [reply src/create-catalog-v001.xml.py:102]: I don't think there's a test written for that case.  `args.in_ttl` is required to have at least one graph passed it, but you're right, the graph could be syntactically valid and have 0 triples in it.

However, if it is empty, then the constructed transitive closure would be empty, making the `catalog-v001.xml` file a "base case" form - no redirects defined.  This seems like a happy-enough outcome to me.

Do you need this tested before you approve?  If so, we'll need to make, test, and merge a follow-on PR for Issue 449 before merging PR 530.

**Review by plbt5 — APPROVED** [review]

**Pull request merged**
